src/agent/onefuzz-agent/src/tasks/coverage/generic.rs

```diff
@@ -362,6 +364,13 @@ impl<'a> TaskContext<'a> {
             .await
             .with_context(|| format!("writing source coverage to {}", path.display()))?;
 
+        let path = self.config.coverage.local_path.join(COBERTURA_COVERAGE_FILE);
+        let cobertura_source_coverage = cobertura(src_coverage)?;
+        let text = &cobertura_source_coverage;
+        fs::write(&path, &text)
+        .await
```

Comment:
Heads up, you'll need to run `cargo fmt` (CI will complain).

Reply:
Just remembered!